src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -145,7 +150,43 @@ def _load_em_from_meid(location, meid, measure):
     return vi_utils.sort_hierarchical_data(data).droplevel("location")
 
 
-# TODO - add project-specific data functions here
+##################
+# Pregnancy Data #
+##################
+
+
+def get_pregnancy_end_rate(location: str) -> pd.DataFrame:
```

Comment:
I haven't heard the term "pregnancy end rate". The docs call it just "pregnancy_incidence" - why the change?

Reply:
this is admittedly a c/p from IV iron--"end" I think here is meant to highlight the fact that what we are getting is not really the "true" pregnancy incidence rate (e.g. new pregnancies), but the incidence rate of the _end_ of pregnancies. I guess "incidence" is implicit in rate. Perhaps `pregnancy_end_incidence` or `end_pregnancy_incidence` would be better?